src/Game/Game.h

```diff
@@ -110,10 +117,11 @@ class Game
     bool _quit = false;
     SDL_Event _event;
     bool _initialized = false;
+    std::unique_ptr<InputController> inputController_;
```

Comment:
I agree with the answer in topic http://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

"Personally I just don't start identifiers with underscores. New addition to my rule: Don't use double underscores anywhere, which is easy as I rarely use underscore."

Yes, single underscore reserved for global namespace and not for class members.
But it's easier to skip using underscore prefix at all then try to remember all the rules.